### PR TITLE
PR to handle Issue #7 and #12

### PR DESCRIPTION
**Pull request by bsieber-mozilla**

### CODE REVIEW

File: src/app/environment.py [hunk 1]
```diff
@@ -38,10 +41,9 @@ def get_version():
...
     This has generic data in repo, but gets the build details in CI.
     """
-    _root = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
-    version_path = os.path.join(_root, "version.json")
     info = {}
-    if os.path.exists(version_path):
-        with open(version_path, "r", encoding="utf8") as version_file:
-            info = json.load(version_file)
+    version_path = Path(__file__).parents[2] / "version.json"
+    print(version_path)
```
Comment (leplatrem) [inline src/app/environment.py:46]: leftover?

File: src/app/environment.py [hunk 2]
```diff
@@ -29,6 +31,7 @@ class Settings(BaseSettings):
 
 @lru_cache()
 def get_settings() -> Settings:
+    """Return the Settings object; use cache"""
```
Comment (leplatrem) [inline src/app/environment.py:34]: nit: some docstrings have newlines, others don't

File: tests/unit/app/test_api.py [hunk 3]
```diff
@@ -18,7 +18,9 @@ def test_read_root(anon_client):
     assert prev_resp.headers["location"] == "./docs"
 
 
-def test_request_summary_is_logged(caplog):
+def test_request_summary_is_logged(
+    caplog,
```
Comment (leplatrem) [inline tests/unit/app/test_api.py:22]: nit: trailing comma?

File: tests/unit/app/test_api.py [hunk 4]
```diff
@@ -18,7 +18,9 @@ def test_read_root(anon_client):
...
+):  # pylint: disable=missing-function-docstring
```
Comment (leplatrem) [inline tests/unit/app/test_api.py:23]: wouldn't there be a way to disable it at the module level (`tests/unit/`?)

**Review by leplatrem — APPROVED** [review]

File: tests/unit/app/test_api.py (hunk 3 above)
Comment (bsieber-mozilla) [reply tests/unit/app/test_api.py:22]: I believe the linter added this upon adjusting the file.

File: tests/unit/app/test_api.py (hunk 4 above)
Comment (bsieber-mozilla) [reply tests/unit/app/test_api.py:23]: Modified pyproject.toml to do so

**Review by leplatrem — APPROVED** [review]
:shipit:

**Pull request merged**